python/ray/serve/_private/router.py

```diff
@@ -28,6 +28,7 @@
 from ray.dag.py_obj_scanner import _PyObjScanner
 from ray.exceptions import RayActorError, RayTaskError
 from ray.util import metrics
+from ray._private.storage import _load_class
```

Comment:
is this `storage` from data team lib?

Reply:
It's common, but I moved it to utils.py for clarity.